python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_pim_sparse_mode.schema.yml

```
@@ -118,3 +118,8 @@ keys:
                         max: 32
                       override:
                         type: bool
+                ssm_range:
+                  type: str
```

```suggestion
                  type: str
                  convert_types:
                    - int
```
router pim sparse-mode
   vrf aaa
      ipv4
         ssm range 220

---

That seems weird to me. ssm range should only be valid multicast IPs, so I would not expect an integer ever. Please try without FastCli.

---

maybe it is an access-list and not the range itself. If so we need to make that clear.

---

yes its(220) access list name